packages/react/src/domain/handles.ts

```diff
@@ -1,12 +1,12 @@
-import { HandleType, FactorOIDC, Handle } from "./types";
+import { HandleType, FactorOIDC, Handle, FactorOTP } from "./types";
 import { Factor } from "@slashid/slashid";
 import {
   getList,
   findFlagByDialCode,
 } from "country-list-with-dial-code-and-flag";
 import { TextConfigKey } from "../components/text/constants";
 
-const FACTORS_WITH_EMAIL = ["webauthn", "email_link"];
+const FACTORS_WITH_EMAIL = ["webauthn", "otp_via_email", "email_link"];
 const FACTORS_WITH_PHONE = ["otp_via_sms", "sms_link"];
```

Comment:
:thinking: IIRC we can do WebAuthn + SMS

Reply:
I think we can but we don't support it for some reason I cannot remember right now 😓